COMMUNICATIONS.md

```diff
@@ -0,0 +1,94 @@
+# OpenSearch Project Communication 
+
+- [Overview](#overview)
+- [Slack](#public-slack)
+  - [Getting Started](#getting-started)
+  - [Workspace Channels](#workspace-channels)
+  - [Tips](#tips)
+
+## Overview
+
+The purpose of this document is to provide references and information regarding
```

Comment:
We don't line-wrap markdown elsewhere because it's ... markdown.

Reply:
I don't think this is an issue? The line wrapping doesn't affect the final markdown output and makes it easier to grok / edit in terminals, VI, and other editors w/ line character limits.

![image](https://user-images.githubusercontent.com/830187/170074702-a128aaca-9160-479c-b3f5-b63e99e86969.png)

Reply:
I am just trying to keep things consistent. This file is formatted differently from other markdown files.